py-scripts/lf_webpage.py

```diff
@@ -1023,7 +1023,7 @@ def graph_2(self, dataset2, lis, bands):
                                           _color_name=['orange'],
                                           _show_bar_value=True,
                                           _enable_csv=True,
-                                          _graph_image_name="Total-url", _color_edge=['black'],
+                                          _graph_image_name="Total-url_http", _color_edge=['black'],
```

Comment:
This is possibly a breaking change for any other automation which goes to grab this graph image. That I am aware, though, we internally are the only consumers of this data. No change requested.